<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtAudioCaptureDlg</class>
 <widget class="QDialog" name="QtAudioCaptureDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>录音</string>
  </property>
  <widget class="QComboBox" name="cbDeviceList">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>19</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>录制设备</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>录制频率</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbRate">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>59</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
  </widget>
  <widget class="QtVolumnBar" name="wgVolumn" native="true">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>102</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
  </widget>
  <widget class="QLabel" name="lbClock">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>97</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>音频时长</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>60</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTime">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>97</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00:000</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbMono">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>单声道</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbStereo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>立体声</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbVolumn">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>97</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>实时音量</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>133</y>
     <width>300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="handleWidth">
    <number>5</number>
   </property>
   <widget class="QPushButton" name="btPause">
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btStart">
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>录制</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btStop">
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btPlay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QtWaveBar" name="wgWave" native="true">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>176</y>
     <width>150</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btOk">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>183</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>保存...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>183</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>cbDeviceList</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>cbRate</zorder>
  <zorder>wgVolumn</zorder>
  <zorder>lbClock</zorder>
  <zorder>label_5</zorder>
  <zorder>lbTime</zorder>
  <zorder>rbMono</zorder>
  <zorder>rbStereo</zorder>
  <zorder>lbVolumn</zorder>
  <zorder>splitter</zorder>
  <zorder>wgWave</zorder>
  <zorder>btOk</zorder>
  <zorder>btCancel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtVolumnBar</class>
   <extends>QWidget</extends>
   <header location="global">gui/QtVolumnBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtWaveBar</class>
   <extends>QWidget</extends>
   <header location="global">gui/QtWaveBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/DataVis.qrc"/>
 </resources>
 <connections/>
</ui>
